Regressions/scripts/propeller/propeller_test.py

```diff
@@ -1,29 +1,25 @@
 # propeller_test.py
 # 
-# Created:  Sep 2014, E. Botero
-# Modified: Feb 2020, M. Clarke  
-#           Sep 2020, M. Clarke 
-#           Oct 2023, Racheal M. Erhard
+# Created:  Oct 2023, Racheal M. Erhard
+# Modified: 
+#           
 
 #----------------------------------------------------------------------
 #   Imports
 # ----------------------------------------------------------------------
 
-import Legacy.trunk.S as SUAVE
-from Legacy.trunk.S.Core import Units
-from RCAIDE.Visualization.Geometry.plot_propeller import plot_propeller
-import matplotlib.pyplot as plt  
-from Legacy.trunk.S.Core import (
-Data, Container,
-)
-
-import numpy as np
-import copy, time
+import RCAIDE
```

Comment:
This should be a new script, and not the same as RCAIDE. Actually change the tests so they're not verbatim to SUAVE.